config/newrelic.yml

```diff
@@ -1,229 +1,694 @@
+  # running as an embedded service within another framework and the agent is
+  # detecting the Sinatra app and skipping the at_exit handler as a result.
+  # Sinatra classically runs the entire application in an at_exit block and would
+  # otherwise misbehave if the Agent's at_exit handler was also installed in
+  # those circumstances. Note: send_data_on_exit should also be set to true in
+  # tandem with this setting.
+  # force_install_exit_handler: false
+
+  # Ordinarily the agent reports dyno names with a trailing dot and process ID
+  # (for example, worker.3). You can remove this trailing data by specifying the
+  # prefixes you want to report without trailing data (for example, worker).
+  # heroku.dyno_name_prefixes_to_shorten: ["scheduler", "run"]
+
+  # If true, the agent uses Heroku dyno names as the hostname.
+  # heroku.use_dyno_names: true
+
+  # If true, enables high security mode. Ensure that you understand the
+  # implication of enabling high security mode before enabling this setting.
+  # https://docs.newrelic.com/docs/agents/manage-apm-agents/configuration/high-security-mode/
+  # high_security: false
+
+  # Configures the hostname for the Trace Observer Host. When configured, enables
+  # tail-based sampling by sending all recorded spans to a Trace Observer for
+  # further sampling decisions, irrespective of any usual agent sampling decision.
+  # infinite_tracing.trace_observer.host: ""
+
+  # Configures the TCP/IP port for the Trace Observer Host
+  # infinite_tracing.trace_observer.port: 443
+
+  # Configure the compression level for data sent to the Trace Observer
+  # May be one of [none|low|medium|high]
+  # By default, compression is not used (level = none)
+  # infinite_tracing.compression_level: none
+
+  # If true, data sent to the Trace Observer will be batched instead of
+  # the default of each span being sent individually
+  # infinite_tracing.batching: false
+
+  # Controls auto-instrumentation of bunny at start up.
+  # May be one of [auto|prepend|chain|disabled].
+  # instrumentation.bunny: auto
+
+  # Controls auto-instrumentation of Curb at start up.
+  # May be one of [auto|prepend|chain|disabled].
+  # instrumentation.curb: auto
+
+  # Controls auto-instrumentation of Delayed Job at start up.
+  # May be one of [auto|prepend|chain|disabled].
+  # instrumentation.delayed_job: auto
+
+  # Controls auto-instrumentation of the elasticsearch library at start up.
+  # May be one of [auto|prepend|chain|disabled].
+  # instrumentation.elasticsearch: auto
+
+  # Controls auto-instrumentation of Excon at start up.
+  # May be one of [enabled|disabled].
+  # instrumentation.excon: auto
+
+  # Controls auto-instrumentation of Grape at start up.
+  # May be one of [auto|prepend|chain|disabled].
+  # Grape, Padrino, and Sinatra aren't supported for Ruby 3.0+
+  # See: https://docs.newrelic.com/docs/apm/agents/ruby-agent/getting-started/ruby-agent-requirements-supported-frameworks/#web_frameworks
```

Comment:
I've had to disable grape instrumentation here because it appears it is now supported for Ruby 3+ according to the documentation linked to above and it was throwing errors in the newrelic logs in staging.  I'll be confirming during acceptance testing whether or not this is going to cause issues with our current monitors.